metadata.json

```diff
@@ -1,15 +1,12 @@
 {
-  "name": "puppetlabs-corosync",
+  "name": "puppet-corosync",
   "version": "0.7.0",
```

Comment:
"because semver" - so you think we're ready to commit to this API? I'm not so sure, I would rather see it mature a bit in p-c.